packages/components/organisms/f-checkout/src/components/_tests/Selector.test.js

```diff
@@ -413,29 +343,181 @@ describe('Selector', () => {
     });
 
     describe('mounted ::', () => {
-        it('should make a call to `initFulfilmentTime` with the available `fulfilmentTimes`', () => {
+        describe('invokes `initFulfilmentTime`', () => {
```

Comment:
I think all these tests are in the wrong section.

On mounted, all we want to check is that `initFullfilmentTime` is invoked with the right parameters.

These tests are actually unit tests of `initFulfilmentTime`.